app/cdap/components/shared/ConfirmDialog/styles.ts

```diff
@@ -35,12 +38,12 @@ export const StyledDialog = styled(Dialog)`
 
 export const StyledAlert = styled(Alert)`
   font-size: 12px;
-  margin-bottom: 8px;
+  margin: 0 -24px 8px -24px;
```

Comment:
Why do we need the negative (horizontal) margin here ?

Reply:
I've moved **Status** component inside the Dialog content area now to adjust the scrolls. 
As MUI dialog content has default margins, had to give as negative to get the required look for Status box. 
Could suggest me if there are any alternatives to this.

Reply:
Understood, thanks